Mathlib/RingTheory/Localization/Module.lean

```diff
@@ -51,11 +58,81 @@ theorem LinearIndependent.localization {ι : Type*} {b : ι → M} (hli : Linear
     refine' Finset.sum_congr rfl fun i hi => _
     rw [← IsScalarTower.algebraMap_smul Rₛ, hg' i hi, smul_assoc]
   refine' (IsLocalization.map_units Rₛ a).mul_right_eq_zero.mp _
-  rw [← Algebra.smul_def, ← map_zero (algebraMap R Rₛ), ← hli, hg' i hi]
+  rw [← Algebra.smul_def, ← _root_.map_zero (algebraMap R Rₛ), ← hli, hg' i hi]
 #align linear_independent.localization LinearIndependent.localization
 
+theorem isLocalizedModule_of_span_eq_top {v : Set M} (hv : span R v = ⊤) :
+    span Rₛ (f '' v) = ⊤ := by
```

Comment:
I'd call this `span_eq_top_of_isLocalizedModule` or `IsLocalizedModule.span_eq_top`.